pkg/instance/handler.go

```diff
@@ -32,6 +32,206 @@ type groupServiceHandler interface {
 	Find(name string) (*model.Group, error)
 }
 
+type SaveDeploymentRequest struct {
+	Name        string `json:"name" binding:"required,dns_rfc1035_label"`
+	Description string `json:"description"`
+	Group       string `json:"group" binding:"required"`
+	TTL         uint   `json:"ttl"`
+	Source      uint   `json:"source"` // TODO: Create from source eg. other deployment
+	Preset      uint   `json:"preset"` // TODO: Create as preset
+}
+
+func (h Handler) SaveDeployment(c *gin.Context) {
+	// swagger:route POST /deployments saveDeployment
+	//
+	// Save a deployment
+	//
+	// Save a deployment...
+	//
+	// Security:
+	//	oauth2:
+	//
+	// responses:
+	//	200: Deployment
+	//	401: Error
+	//	403: Error
+	//	404: Error
+	//	415: Error
+	var request SaveDeploymentRequest
+	if err := handler.DataBinder(c, &request); err != nil {
+		_ = c.Error(err)
+		return
+	}
+
+	group, err := h.groupService.Find(request.Group)
```

Comment:
Should this already check if the user has access to the group?
I think we do check it in some way in `handler.CanWriteDeployment` 🤔 but not as explicitly